### ISSUE TITLE
The calculation method of gene frequency (or copy) in RGI_Heatmap

### ISSUE THREAD

**Issue by ZYX593889540** [arpcard/rgi#277]
How is the frequency or copy number of resistance genes calculated for each sample in the obtained output table, and based on which screening is made.

**Comment by agmcarthur** [comment]
Documentation found here: https://github.com/arpcard/rgi/blob/master/docs/rgi_main.rst

Currently, frequency of ARGs within samples is not supported. Yellow represents a Perfect annotation, teal represents a Strict annotation, purple represents no ARG found.

The _--frequency_ flag will place a histogram along the top of the figure, showing how many isolates share the same resistome profile.

For example, this is a heat map from a number of isolate genomes, where I generated RGI json output for each genome. I used the flags _--frequency --cluster both_.

![cluster_both_frequency-39](https://github.com/arpcard/rgi/assets/27361102/a21333c5-6e28-4bf7-91cf-2f4b5994c179)
